.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,6 +9,7 @@
 - [ ] The code is mine or it's from somewhere with an MIT-compatible license.
 - [ ] The code is efficient, to the best of my ability, and does not waste computer resources.
 - [ ] The code is stable and I have tested it myself, to the best of my abilities.
+- [ ] If the code introduces new aliases, I provide a valid use case for all plugin users down below.
```

Comment:
@carlosala would this work?
```suggestion
- [ ] If the code introduces new aliases, I provide a valid use case for all plugin users. See CONTRIBUTING file.
```

Reply:
Perfect for me! 👌🏻

Reply:
> @carlosala would this work?
> 
> ```suggestion
> 
> - [ ] If the code introduces new aliases, I provide a valid use case for all plugin users. See CONTRIBUTING file.
> 
> ```

Just a note from me. Could we link to the file itself?

Reply:
The problem is that when you are writing the PR you have the raw display of markdown, and it's going to be a bit shitty to have that `[foo](https://bar.com)` syntax.

Reply:
Ahh very true. Didn't take that into consideration I was thinking about it being rendered for the end user.

Reply:
looks like we can merge @mcornella